Cargo.toml

```diff
@@ -18,8 +18,8 @@ celestia-tendermint-proto = "0.32.1"
 # Uncomment to apply local changes
 #beetswap = { path = "../beetswap" }
 #blockstore = { path = "../blockstore" }
-#celestia-tendermint = { path = "../celestia-tendermint-rs/tendermint" }
```

Comment:
this needs to be removed when https://github.com/eigerco/celestia-tendermint-rs/pull/31 is merged